README.md

```diff
@@ -80,12 +80,32 @@ resource:
       pod: true
 ```
 
+### Working with RBAC
+
+Kubernetes Engine clusters running versions 1.6 or higher introduced Role-Based Access Control (RBAC). We can create `ServiceAccount` for it to work with RBAC.
+
+```console
+$ kubectl create -f kubewatch-service-account.yaml
+```
+
+If you do not have permission to create it, you need to become a admin first. Do this:
+
+```
+$ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=REPLACE_EMAIL_HERE
+```
+
+Append `serviceAccountName: kubewatch` below `spec:` in `kubewatch.yaml`. Then created again with:
```

Comment:
Suggest rewording it a bit, also providing a cmdline to do tthis edit, e.g.:

Edit  `kubewatch.yaml`, and create a new field under `spec` with `serviceAccountName: kubewatch`, you can achieve tihs by running:
```console
$ sed -i '/spec:/a\ \ serviceAccountName: kubewatch' kubewatch.yaml
```